src/api/app/views/webui/request/beta_show_tabs/_review_summary.html.haml

```diff
@@ -0,0 +1,38 @@
+:ruby
+  review_by = if review.by_user
```

Comment:
We keep doing this over and over again. In this PR alone here and also in [bs_request_overview_avatars_component.html.haml](https://github.com/openSUSE/open-build-service/pull/12863/files#diff-fa44a752f6648ecb8798e93aeeaafd8d77197e4bb05477bc6d6531a55222031bR6-R9)

I should be able to ask `Review` if it's for a project or for a package without having to cover for the case that it has both `by_package` and `by_project` for a `by_package`.